exercises/book-store/canonical-data.json

```diff
@@ -85,6 +85,15 @@
          },
          "expected": 5120
         },
+        {
+         "property": "total",
+         "description": "Two groups of four is cheaper than group of five and three",
```

Comment:
I think `than group` should be `than groups`

Reply:
Oops. Will fix tonight

Reply:
The test case above this one has the same grammatical mistake.  It should probably be fixed in a separate PR and would bump version by `?.?.1`.

Reply:
OK, fixed this here, but not in the other test case.

Reply:
That's good, it should be fixed in a separate PR.